plugin/evm/vm_test.go

```diff
@@ -477,3 +480,53 @@ func TestConfigureLogLevel(t *testing.T) {
 		})
 	}
 }
+
+func TestToSet(t *testing.T) {
```

Comment:
In this test we're also missing a subtest for the case where `set.AddWeight` fails

Reply:
Not sure when this function returns error though https://github.com/flare-foundation/flare/blob/master/snow/validators/set.go#L205
:thinking: